Probesys/bileto
# new: Upload images to the server with TinyMCE

## Related issue(s)



https://github.com/Probesys/bileto/issues/149

## Changes



- Add a `MessageDocument` entity to store a references to the documents
- Add a `MessageDocumentStorage` service to manipulate `MessageDocument`s on the filesystem
- Add two endpoints to upload and download/see documents
- Configure TinyMCE to upload images to the upload endpoint

## How to test manually



- open or create a ticket
- in the message, copy/paste an image
- also, upload an image via the image button
- post the message
- check that both images appear in the final message

## Checklist
